### PR TITLE
Steam Test Server - Update 14

### PR DESCRIPTION
**Pull request by Sunaries**
Improvements in preparation for the Open Beta.

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge.

File: EnglishTranslation-master/ITEM.tsv [hunk 1]
```diff
@@ -9573,7 +9573,7 @@ ITEM_20151224_009572	Officer's Journal Part 2
 ITEM_20151224_009573	The second part written by an officer of the Delmore Fortress regarding his work. Readable
 ITEM_20151224_009574	Withered Plant
 ITEM_20151224_009575	Warpstone
-ITEM_20151224_009576	Allows you to move to a designated location within your current area. Right-click to use.{nl}Cooldown of 1 hour.
+ITEM_20151224_009576	Warps you to the beginning of the area you are currently in. Right-click to use.{nl}Cooldown of 1 hour.
 ITEM_20151224_009577	One of the task requirements by the Hoplite Submaster. It looks extremely sturdy.
 ITEM_20151224_009578	One of the task requirements by the Hoplite Submaster. It would be fatal to be stabbed by it.
 ITEM_20151224_009579	One of the task requirements by the Hoplite Submaster. It scatters at the slightest breath
```
Comment (sususuyo) [inline EnglishTranslation-master/ITEM.tsv]: you didn't touch this line but should there be a period at the end of this too?

File: EnglishTranslation-master/SKILL.tsv [hunk 2]
```diff
@@ -3266,7 +3266,7 @@ SKILL_20150401_003265	Attack #{SkillAtkAdd}#{nl}Install #{CaptionRatio}#
 SKILL_20150401_003266	Evasion - #{CaptionRatio}# {nl}Pointing AI duration 10 minutes
 SKILL_20150401_003267	Detect hidden monster{nl}Hounding AI duration 10 minutes
 SKILL_20150401_003268	Companion guards against monsters approaching.
-SKILL_20150401_003269	{#339999}{ol}[Fear]{/}{/} Duration #{CaptionRatio}# seconds{nl}If distance between player and companion is at least {nl}250 meters, disable AI{nl}Growling AI Duration 10 minutes
+SKILL_20150401_003269	{#339999}{ol}[Fear]{/}{/} Duration: #{CaptionRatio}# seconds{nl}Disables AI if the distance is{nl}at least 250 between you and the companion{nl}Growling AI Duration: 10 minutes
```
Comment (sususuyo) [inline EnglishTranslation-master/SKILL.tsv]: should there be a measurement unit after 250?

File: EnglishTranslation-master/SKILL.tsv [hunk 3]
```diff
@@ -4989,7 +4989,7 @@ SKILL_20150729_004988	Heal: Remove Damage
 SKILL_20150729_004989	[Heal] will not inflict damage and react to enemies.
 SKILL_20150729_004990	Increases the range of [Melstis] to 80.
 SKILL_20150729_004991	Increases the maximum number of zombies from [Zombify] by 1.
-SKILL_20150730_004992	Summons a devil from the card placed in the Grimoire. The attack and defense of the summoned devil is affected by your INT and SPR. Increases the abilities by 10% proportional to the amount of stars of the card. Only Devil-type cards can be placed in the Grimoire.
+SKILL_20150730_004992	Summon a devil from the card placed in the Grimoire. The attack and defense of the summoned devil is affected by your INT and SPR. Increases the stats by 10% proportional to the amount of stars of the card. Only Devil-type cards can be placed in the Grimoire.
```
Comment (sususuyo) [inline EnglishTranslation-master/SKILL.tsv]: I can't really tell how the stats are increasing... is it 10%(multiplied by # of stars) of your INT and SPR?
If so, maybe the description should be
`The attack and defense of the summoned devil is affected by 10% your INT and SPR, proportional to the amount of stars of the card.`
But if it means the stats in general are increased by the stars on the card, then maybe all that should be changed is
 `The stats increase by 10% proportional to the amount of stars of the card.`

If you do make a change here, it should probably apply to the Shaggoth description as well

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Pull request merged**